### PR TITLE
fix URL encoding for wikipedia references in filters.rank.entropy and filters.rank.shannon_entropy docstring

### PR DESCRIPTION
**Pull request by LuisFF**
## Checklist

- [ ] [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- [ ] Gallery example in `./doc/examples` (new features only)
- [ ] Benchmark in `./benchmarks`, if your changes aren't covered by an
  existing benchmark
- [ ] Unit tests
- [ ] Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)

## For reviewers

- [ ] Check that the PR title is short, concise, and will make sense 1 year
  later.
- [ ] Check that new functions are imported in corresponding `__init__.py`.
- [ ] Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.
- [ ] Consider backporting the PR with `@meeseeksdev backport to v0.14.x`

### CODE REVIEW

**Comment by hmaarrfk** [comment]
Hmm, so it seems that restructred text doesn't include the closing braket in the url. not sure why.

**Comment by hmaarrfk** [comment]
Is there a way to add "brackets" in rst that specify that the url isn't completed yet? I'm thinking `<url>_` maybe?

File: skimage/filters/rank/generic.py [hunk 1]
```diff
@@ -947,7 +947,7 @@ def entropy(image, selem, out=None, mask=None, shift_x=False, shift_y=False):
 
     References
     ----------
-    .. [1] https://en.wikipedia.org/wiki/Entropy_(information_theory)
+    .. [1] https://en.wikipedia.org/wiki/Entropy_%28information_theory%29
```
Comment (hmaarrfk) [inline skimage/filters/rank/generic.py]:
```suggestion
    .. [1] `https://en.wikipedia.org/wiki/Entropy_(information_theory) <https://en.wikipedia.org/wiki/Entropy_(information_theory)>`_
```

File: skimage/measure/entropy.py [hunk 2]
```diff
@@ -26,7 +26,7 @@ def shannon_entropy(image, base=2):
 
     References
     ----------
-    .. [1] https://en.wikipedia.org/wiki/Entropy_(information_theory)
+    .. [1] https://en.wikipedia.org/wiki/Entropy_%28information_theory%29
```
Comment (hmaarrfk) [inline skimage/measure/entropy.py]:
```suggestion
    .. [1] `https://en.wikipedia.org/wiki/Entropy_(information_theory) <https://en.wikipedia.org/wiki/Entropy_(information_theory)>`_
```

**Comment by hmaarrfk** [comment]
I think it is a small change, but helps ensure readability. What do you think? I'll approve either change, and some other core member can come in to pull.

**Comment by sciunto** [comment]
@LuisFF Can you commit the changes proposed by @hmaarrfk please (use the button on github)? Then, it will be ready to go.

**Comment by sciunto** [comment]
@LuisFF I'm sorry to ping you again. Can you take a minute to merge the proposed changes? Otherwise, we will have to open a new PR to ingrate this fix in the next release

Thanks!

**Comment by LuisFF** [comment]
@sciunto I would love to merge, but there is one test that keeps failing and I believe it's not related to this changes

**Comment by sciunto** [comment]
No worries about the travis failure, it's not related to your contribution. You can go ahead.

**Comment by sciunto** [comment]
In case my request is not clear...

fix URL encoding for wikipedia references in filters rank entropy and filters rank shannon_entropy docstring by LuisFF · Pull Request 4007 · scikit-imagescikit-image

**Comment by LuisFF** [comment]
@sciunto thanks for the request hint

**Comment by sciunto** [comment]
Perfect! Thank you.

**Pull request merged**